### ISSUE TITLE
? labeled with - lemma in train-s592

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/UD_Hindi-HDTB#1]
```
# sent_id = train-s592
# text = कैसे पहुँचें?
1       कैसे      कैसे      PRON    WQ      PronType=Int    2       obl     _       ChunkId=RBP|ChunkType=head|Translit=kaise
2       पहुँचें     पहुँच     VERB    VM      Mood=Sub|VerbForm=Fin|Voice=Act 0       root    _       SpaceAfter=No|Vib=एं|Tam=eM|ChunkId=VGF|ChunkType=head|Stype=int\
errogative|Translit=pahum̃ceṁ
3       ?       -       PUNCT   SYM     _       2       punct   _       ChunkId=BLK|ChunkType=head|Translit=?
```

The `-` lemma looks incorrect to me, not that I actually know anything about Hindi

**Comment by dan-zeman** [comment]
Thanks! Fixed in dev (https://github.com/UniversalDependencies/UD_Hindi-HDTB/commit/1055b5e9a9748fd553aac0a65f025126b590b9b4).

**Comment by AngledLuffa** [comment]
Thanks!  Timeline for fixes like that is 2.8 in May?

**Comment by AngledLuffa** [comment]
Also, there is a similar instance in dev which I had not thought to look for previously:

```
# sent_id = dev-s67
# text = कैसे पहुँचें?
1       कैसे      कैसे      PRON    WQ      PronType=Int    2       obl     _       ChunkId=RBP|ChunkType=head|Translit=kaise
2       पहुँचें     पहुँच     VERB    VM      Mood=Sub|Number=Plur|Person=3|VerbForm=Fin|Voice=Act    0       root    _       SpaceAfter=No|Vib=एं|Tam=eM|ChunkId=VGF|ChunkType=head|Stype=interrogativ\
e|Translit=pahum̃ceṁ
3       ?       -       PUNCT   SYM     _       2       punct   _       ChunkId=BLK|ChunkType=head|Translit=?
```

(dev dataset, not dev branch)

**Comment by dan-zeman** [comment]
Both should be covered by the fix I did. And yes, fixes done now will become part of release 2.8 in May.
